### PR TITLE
Changed POTW link (Issue #21)

### PR DESCRIPTION
**Pull request by arron-tij**
Changed the POTW link from timeline section according to Issue #21 and also from the projects section.
I have used relative links.

### CODE REVIEW

File: _data/index/careers.yml [hunk 1]
```diff
@@ -122,7 +122,7 @@
     detail: Summer. 2017
     i18n: company_b_date
   job:
-    detail: <a target="_blank" href="https://docs.google.com/document/d/1J2c5zkhvZQ8U2S5PNxZoTQce0MO2JCdPOrnfViZ2Lv8/edit?usp=sharing"><h1><b>POTW</b></h1></a>
+    detail: <a target="_blank" href="../resources/potw.html"><h1><b>POTW</b></h1></a>
```
Comment (shivanshs9) [inline _data/index/careers.yml]: I would suggest using root-relative links.

File: _data/index/projects.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 - name: Problem of the Week
   i18n: google
-  url: https://docs.google.com/document/d/1J2c5zkhvZQ8U2S5PNxZoTQce0MO2JCdPOrnfViZ2Lv8/edit?usp=sharing
+  url:  /resources/potw.html
```
Comment (shivanshs9) [inline _data/index/projects.yml]: Also, check the link using in Navbar. We don't use 'html' suffix in links.

**Review by shivanshs9 — CHANGES_REQUESTED** [review]

File: _data/index/projects.yml (hunk 2 above)
Comment (arron-tij) [reply _data/index/projects.yml]: Made changes according to both the above reviews and squashed those into one.

**Review by shivanshs9 — APPROVED** [review]
Thanks for your contributions! It looks fine to me. 👍

**Pull request merged**